frontend/src/components/Overlay/OverlayNav.css

```diff
@@ -0,0 +1,92 @@
+.overlay-nav {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	gap: 1rem;
+	height: 5rem;
+	padding: 0;
+	border-bottom-right-radius: 0.625rem;
+	border-bottom-left-radius: 0.625rem;
```

Comment:
Again, does this footer region overflow the overlay itself, or why did you need explicit radius here?

Reply:
The OverlayNav is not part of the Overlay component, so I do think this is needed, but happy to change it if there's a better way? 
Without: 
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/e333f2c1-c1ad-4e99-ad9a-574363860261)

Reply:
Right that makes sense!